fontforgeexe/charinfo.c

```diff
@@ -3659,28 +3659,43 @@ static int CI_DefLCChange(GGadget *g, GEvent *e) {
 return( true );
 }
 
+struct ParsedDecomp_s {
+    unichar_t* decomp;
+    bool truncated;
+};
+
+typedef struct ParsedDecomp_s ParsedDecomp;
+
```

Comment:
Personal preference, but I don't think defining a struct for this is necessary. I'd do something like

```c
int CI_ParseUserDecomposition(char *inp, unichar_t **decomp) {
    *decomp = malloc(blah);
    ....
    return truncated;
}
```

Reply:
Probably just Rust influence on me :) I prefer this way, but will change it if you insist.